Experimental/MultiplayerUseCases/Assets/UseCaseSamples/NetvarVsRpc/Scripts/ColorManager.cs

```diff
@@ -65,23 +65,23 @@ void Update()
 
```

Comment:
Another nitpick too: abbreviation-wise, we should rename all usages of RPC -> Rpc in comments, assets, and in namespaces too.